index.js

```diff
@@ -26,7 +26,7 @@ export default class CrossfadeImage extends Component {
             if (!this.timeout) clearTimeout(this.timeout);
             this.timeout = setTimeout(
               () => this.setState({ bottomOpacity: 0 }),
-              10
+              50
```

Comment:
20 ms seems fair to me. I aimed to 50 ms to have enough margin, but if your tests are good with 20ms, that's great.